wikipendium/wiki/templates/article.html

```diff
@@ -15,10 +15,13 @@
 {% endblock %}
 
 {% block content %}
+{% autoescape off %}
+    {{ toc }}
+{% endautoescape %}
+<a href="javascript:;" id="toggle-toc">‹</a>
```

Comment:
If something is "back in style", it means that it used to be popular ("in style"), but then became unpopular ("fell out of style"), but then became popular again ("back in style").

I was under the impression that doing `href=javascript:;` was currently out of style, hence my question on whether or not it is now back in style.